<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1119</width>
    <height>707</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NerDisco 0.5b</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/resources/icon.ico</normaloff>:/resources/icon.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="2,1">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0,0">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="Deck" name="widgetDeckA" native="true"/>
        </item>
        <item>
         <widget class="Deck" name="widgetDeckB" native="true"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QSlider" name="horizontalSliderCrossfade">
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksBothSides</enum>
        </property>
        <property name="tickInterval">
         <number>10</number>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="topMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Crossfade</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignHCenter|Qt::AlignTop</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>Audio</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QLabel" name="labelSpectrumImage">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_4">
        <property name="title">
         <string>LED display</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0">
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <layout class="QGridLayout" name="gridLayout">
             <item row="1" column="1">
              <widget class="QLabel" name="labelRealImage">
               <property name="text">
                <string/>
               </property>
               <property name="scaledContents">
                <bool>true</bool>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="labelFinalImage">
               <property name="text">
                <string notr="true"/>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>LED image</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignBottom|Qt::AlignHCenter</set>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Combined image</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignBottom|Qt::AlignHCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QGridLayout" name="gridLayout_2">
             <property name="topMargin">
              <number>0</number>
             </property>
             <item row="1" column="1">
              <layout class="QHBoxLayout" name="horizontalLayout_9">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QLabel" name="label_17">
                 <property name="text">
                  <string>-1</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_8">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QLabel" name="label_18">
                 <property name="text">
                  <string>+1</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="3" column="1">
              <layout class="QHBoxLayout" name="horizontalLayout_8">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QLabel" name="label_15">
                 <property name="text">
                  <string>0.5</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_7">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QLabel" name="label_16">
                 <property name="text">
                  <string>2</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="5" column="1">
              <layout class="QHBoxLayout" name="horizontalLayout_7">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QLabel" name="label_13">
                 <property name="text">
                  <string>1.5</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_6">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QLabel" name="label_14">
                 <property name="text">
                  <string>3.5</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="4" column="1">
              <widget class="QSlider" name="horizontalSliderGamma">
               <property name="minimum">
                <number>150</number>
               </property>
               <property name="maximum">
                <number>350</number>
               </property>
               <property name="pageStep">
                <number>10</number>
               </property>
               <property name="value">
                <number>220</number>
               </property>
               <property name="sliderPosition">
                <number>220</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="invertedAppearance">
                <bool>false</bool>
               </property>
               <property name="invertedControls">
                <bool>false</bool>
               </property>
               <property name="tickPosition">
                <enum>QSlider::TicksBelow</enum>
               </property>
               <property name="tickInterval">
                <number>10</number>
               </property>
              </widget>
             </item>
             <item row="4" column="0">
              <widget class="QLabel" name="label_12">
               <property name="text">
                <string>Display gamma</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QSlider" name="horizontalSliderBrightness">
               <property name="minimum">
                <number>-50</number>
               </property>
               <property name="maximum">
                <number>50</number>
               </property>
               <property name="pageStep">
                <number>10</number>
               </property>
               <property name="value">
                <number>0</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="invertedAppearance">
                <bool>false</bool>
               </property>
               <property name="invertedControls">
                <bool>false</bool>
               </property>
               <property name="tickPosition">
                <enum>QSlider::TicksBelow</enum>
               </property>
               <property name="tickInterval">
                <number>10</number>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Contrast</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QSlider" name="horizontalSliderContrast">
               <property name="minimum">
                <number>-50</number>
               </property>
               <property name="maximum">
                <number>50</number>
               </property>
               <property name="pageStep">
                <number>10</number>
               </property>
               <property name="value">
                <number>0</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="invertedAppearance">
                <bool>false</bool>
               </property>
               <property name="invertedControls">
                <bool>false</bool>
               </property>
               <property name="tickPosition">
                <enum>QSlider::TicksBelow</enum>
               </property>
               <property name="tickInterval">
                <number>10</number>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>Brightness</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1119</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDeckA">
    <property name="title">
     <string>DeckA</string>
    </property>
    <addaction name="actionLoadDeckA"/>
    <addaction name="actionSaveDeckA"/>
    <addaction name="actionSaveAsDeckA"/>
   </widget>
   <widget class="QMenu" name="menuDeckB">
    <property name="title">
     <string>DeckB</string>
    </property>
    <addaction name="actionLoadDeckB"/>
    <addaction name="actionSaveDeckB"/>
    <addaction name="actionSaveAsDeckB"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Datei</string>
    </property>
    <addaction name="actionOptions"/>
    <addaction name="actionBeenden"/>
   </widget>
   <widget class="QMenu" name="menuAudio">
    <property name="title">
     <string>Audio</string>
    </property>
    <addaction name="actionAudioDevices"/>
    <addaction name="actionAudioRecord"/>
    <addaction name="actionAudioStop"/>
   </widget>
   <widget class="QMenu" name="menuDisplay">
    <property name="title">
     <string>Display</string>
    </property>
    <addaction name="actionDisplayStart"/>
    <addaction name="actionDisplayStop"/>
   </widget>
   <addaction name="menuSettings"/>
   <addaction name="menuDeckA"/>
   <addaction name="menuDeckB"/>
   <addaction name="menuAudio"/>
   <addaction name="menuDisplay"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOptions">
   <property name="text">
    <string>Options...</string>
   </property>
  </action>
  <action name="actionBeenden">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionLoadDeckA">
   <property name="text">
    <string>Load into deck</string>
   </property>
  </action>
  <action name="actionLoadDeckB">
   <property name="text">
    <string>Load into deck</string>
   </property>
  </action>
  <action name="actionSaveDeckA">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSaveAsDeckA">
   <property name="text">
    <string>Save as...</string>
   </property>
  </action>
  <action name="actionSaveDeckB">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSaveAsDeckB">
   <property name="text">
    <string>Save as...</string>
   </property>
  </action>
  <action name="actionAudioRecord">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Record</string>
   </property>
  </action>
  <action name="actionAudioStop">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </action>
  <action name="actionDisplayStart">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </action>
  <action name="actionDisplayStop">
   <property name="text">
    <string>Stop</string>
   </property>
  </action>
  <action name="actionAudioDevices">
   <property name="text">
    <string>Device</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Deck</class>
   <extends>QWidget</extends>
   <header>Deck.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
